app/assets/javascripts/metadata.js

```diff
@@ -0,0 +1,37 @@
+(function (global) {
+  var MetadataDetails;
+
+  MetadataDetails = {
+    init: function (el) {
+      var _this = this;
+      _this.addButton(el);
+      _this.toggleButton(el);
+    },
+
+    addButton: function(el) {
+      var button = "<button type='button' class='btn btn-default btn-details' data-toggle='collapse' data-target='#metadataDetails' aria-expanded='false' aria-controls='metadataDetails'><span class='btn-text'>More details</span><span class='btn-caret caret-down'>>></span></button>";
```

Comment:
Is it better to inject the button HTML with JS or include it with the erb template and hide it with CSS? 🤔

Reply:
I'd vote to do it in the ERB template; if nothing else, it'll be easier to add i18n strings for it.